src/Kernel32.Shared/Win32Exception.cs

```diff
@@ -46,7 +46,7 @@ public Win32Exception()
         /// <param name="error">The Win32 error code associated with this exception.</param>
         public Win32Exception(Win32ErrorCode error)
 #if DESKTOP
-            : base((int)error)
+            : this(error, error.GetMessage() ?? $"Unknown error (0x{(int)error:x8})")
```

Comment:
I have checked here at work and I have both en-US and de-DE packages on Windows 10 and on .NET 4.6.2 installed. It seems that .NET delegates to the OS on these cases and the OS returns the current language despite the fact that we have other packages installed on the machine.